src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/RecoveryServicesBackupManagement.csproj

```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{14F715EC-C03D-4BB4-83BD-6EEFAA6B0C67}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.Azure.Management.RecoveryServicesBackupManagement</RootNamespace>
+    <RootNamespace>Microsoft.Azure.Management.RecoveryServices.Backup</RootNamespace>
```

Comment:
no .. this is only for newly added files. Verified both powershell and Portal with new dll and they compiled fine.